.github/workflows/e2e-tests-workflow.yml

```diff
@@ -66,14 +68,14 @@ jobs:
           plugin_version=$(node -pe "require('./package.json').version")
           echo plugin version: $plugin_version
           echo plugin name: $AD_OPENSEARCH_DASHBOARDS_PLUGIN_NAME
-          if docker pull $OPENSEARCH_DOCKER_IMAGE:latest
+          if docker pull $OPENSEARCH_DOCKER_IMAGE:$OPENSEARCH_DOCKER_TAG
```

Comment:
The `latest` will not point to `1.0.0-rc1`?

Reply:
It's a good point, the digests do match on docker hub for `latest` and `1.0.0-rc1`. The reason I was thinking of explicitly setting it to `1.0.0-rc1`, was to treat it as pre-release testing, and just be more explicit/verbose. Whatever is tagged as `1.0.0-rc1` will be the one eventually promoted to the release docker image, and is the one we want to be sure our plugin is compatible with.

Let me know what you think.

Reply:
Got it, that's ok. We can move to latest later to make sure all test can pass on latest version.

Reply:
Not sure if backward compatibility is required for future OpenSearch versions. We may need to add multiple integration test workflows to run on old versions to test if latest code can work on all OpenSearch versions.

Reply:
Since plugins are still required to have the same version as core, I think this is not a problem for now, since new plugin versions currently can't be installed on core versions that are mismatched.